### ISSUE TITLE
Pantheon DE: dock disappears when on-screen keyboard is used

### ISSUE THREAD

**Issue by davidak** [NixOS/nixpkgs#88813]
**Describe the bug**

i want to use the dock but it disappears

https://peertube.social/videos/watch/5ca1b440-6684-4b19-a2fe-0aea2bb949d6

**To Reproduce**
1. enable on-screen keyboard in settings
2. try to use dock

**Expected behavior**
dock still usable

not sure how elementary OS handles it

**Notify maintainers**
@worldofpeace 

**Metadata**
 - system: `"x86_64-linux"`
 - host os: `Linux 5.6.13, NixOS, 20.03.1896.025deb80b24 (Markhor)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.3.4`
 - channels(root): `"home-manager-20.03, nixos-20.03.1896.025deb80b24, nixos-hardware, nixos-unstable-20.09pre226012.b47873026c7"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`

**Comment by jtojnar** [comment]
Possible upstream issue: https://bugs.launchpad.net/onboard/+bug/1261317

**Comment by davidak** [comment]
Not sure if it is https://github.com/elementary/keyboard

**Comment by jtojnar** [comment]
Visually, it looks more like onboard. Try the menu button below <kbd>1 2 3</kbd>

**Comment by davidak** [comment]
Yes, there it says "Onboard Home". Not sure what their plan is with that touch keyboard. Maybe it will replace Onboard and fix this issue.

**Comment by jtojnar** [comment]
I think that is abandoned, or at least should be now that caribou is abandoned and touch keyboard is integrated into mutter (which they use).
